tests/unit/test_steps_utils.py

```diff
@@ -51,3 +51,10 @@ def test_arn_builder_sagemaker_wait_completion():
                                       IntegrationPattern.WaitForCompletion)
     assert arn == "arn:aws:states:::sagemaker:createTrainingJob.sync"
 
+
+def test_merge_dicts():
+    d1 = {'a': {'aa': 1, 'bb': 2, 'cc': 3}, 'b': 1}
+    d2 = {'a': {'bb': {'aaa': 1, 'bbb': 2}}, 'b': 2, 'c': 3}
+
+    merge_dicts(d1, d2, 'd1', 'd2')
+    assert d1 == {'a': {'aa': 1, 'bb': {'aaa': 1, 'bbb': 2}, 'cc': 3}, 'b': 2, 'c': 3}
```

Comment:
nit: indent the dicts to make it easier to read

Reply:
Always for improving code readability - changes will be included in the next commit